backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -487,13 +607,33 @@ def _create_ingest_pipeline(self) -> None:
             body={
                 "description": "Extract attachment information",
                 "processors": [
+                    {"remove": {"field": FILE_PIPELINE_SOURCE_FIELD}},
                     {
                         "attachment": {
                             "field": FILE_PIPELINE_SOURCE_FIELD,
                             "target_field": FILE_PIPELINE_DESTINATION_FIELD,
                         }
                     },
-                    {"remove": {"field": FILE_PIPELINE_SOURCE_FIELD}},
+                    {
+                        "set": {
+                            "if": "ctx.file_data.language == 'fr'",
```

Comment:
Warning ! 
`FILE_PIPELINE_SOURCE_FIELD` should match `file_data` here to work as expected.